sources/Tools/DatasetHandler.py

```diff
@@ -113,10 +118,17 @@ def __init__(self, parse: bool = False, train: bool = False):
         Public methods
     """
 
-    def save_header(self, header_file: str = default_header_file):
+    def save_header(self, header_file: str = default_header_files["advanced"]):
         self._save_npy(header_file, self.analysed_header)
 
-    def load_header(self, header_file: str = default_header_file):
+    def load_header(self):
+        # test if not crash with all program
+        header_file = self.default_header_files[self.header_visualizer]
+        print(self.header_visualizer)
+        print(self.func_visualizer)
```

Comment:
delete the print please

Reply:
I was using it to do scatter plot, i can remove now